### PR TITLE
11075: images from Base64 string were not displayed

### PR DESCRIPTION
**Pull request by savvinsergey**

### CODE REVIEW

File: test/visualTest.ts [hunk 1]
```diff
@@ -1239,6 +1239,25 @@ module powerbi.extensibility.visual.test {
...
+                dataViewBuilder.valuesImage[1] = dataImage;
+
+                let chicletSlicerConverter: ChicletSlicerConverter = new ChicletSlicerConverter(
+                    dataViewBuilder.getDataView(),
+                    visualBuilder.visualHost);
+
+                chicletSlicerConverter.convert();
+
+                expect(chicletSlicerConverter.dataPoints[0].imageURL).toBe(linkToImage);
+                expect(chicletSlicerConverter.dataPoints[1].imageURL).toBe(dataImage);
```
Comment (ignatvilesov) [inline test/visualTest.ts]: There's should a single ```expect``` statement per ```it```. Please split up this it into two ones.

Comment (savvinsergey) [inline test/visualTest.ts]: Done

File: test/visualTest.ts [hunk 2]
```diff
@@ -1239,6 +1239,42 @@ module powerbi.extensibility.visual.test {
                         expect(dataPoint.imageURL).not.toBe(firstUrl);
                     });
             });
+
+            it("imageURL after convert", () => {
+
+                describe("imageURL mustn't have 'undefined' value", () => {
+                    const dataViewBuilder: ChicletSlicerData = new ChicletSlicerData();
+                    beforeEach(() => {
+                        dataViewBuilder.valuesImage = new Array(1);
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Why do we need this?

File: test/visualTest.ts [hunk 3]
```diff
@@ -1239,6 +1239,42 @@ module powerbi.extensibility.visual.test {
...
+                    });
+                    it("image value is link", () => {
+                        const linkToImage: string = dataViewBuilder.valuesImage[0];
+                        dataViewBuilder.valuesImage[0] = linkToImage;
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Why do we need this? You just reassigned the same value.

File: test/visualTest.ts [hunk 4]
```diff
@@ -1239,6 +1239,42 @@ module powerbi.extensibility.visual.test {
...
+                        let chicletSlicerConverter: ChicletSlicerConverter = new ChicletSlicerConverter(
+                            dataViewBuilder.getDataView(),
+                            visualBuilder.visualHost);
+
+                        chicletSlicerConverter.convert();
+
+                        expect(chicletSlicerConverter.dataPoints[0].imageURL).toBe(linkToImage);
+                    });
+
+                    it("image value is base64 image", () => {
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Source code of tho its are the same. Can we move the common code to a function?

Comment (savvinsergey) [inline test/visualTest.ts]: Done

File: test/visualTest.ts (hunk 3 above)
Comment (savvinsergey) [inline test/visualTest.ts]: Done

File: test/visualTest.ts (hunk 2 above)
Comment (savvinsergey) [inline test/visualTest.ts]: Done

File: test/visualTest.ts [hunk 5]
```diff
@@ -1239,6 +1239,39 @@ module powerbi.extensibility.visual.test {
                         expect(dataPoint.imageURL).not.toBe(firstUrl);
                     });
             });
+
+            it("imageURL after convert", () => {
+
+                describe("imageURL mustn't have 'undefined' value", () => {
+                    const dataViewBuilder: ChicletSlicerData = new ChicletSlicerData();
```
Comment (ignatvilesov) [inline test/visualTest.ts]: It should be in ```forEach``` statement.

File: test/visualTest.ts [hunk 6]
```diff
@@ -1239,6 +1239,39 @@ module powerbi.extensibility.visual.test {
...
+                    dataViewBuilder.valuesImage = dataViewBuilder.valuesImage.slice(0, 1);
```
Comment (ignatvilesov) [inline test/visualTest.ts]: It should be in ```forEach``` statement.

Comment (savvinsergey) [inline test/visualTest.ts]: Done

File: test/visualTest.ts (hunk 5 above)
Comment (savvinsergey) [inline test/visualTest.ts]: Done

File: test/visualTest.ts [hunk 7]
```diff
@@ -1239,6 +1239,41 @@ module powerbi.extensibility.visual.test {
                         expect(dataPoint.imageURL).not.toBe(firstUrl);
                     });
             });
+
+            it("imageURL after convert", () => {
```
Comment (ignatvilesov) [inline test/visualTest.ts]: Please use ```describe``` instead of ```it``` here.

Comment (savvinsergey) [inline test/visualTest.ts]: Done

**Pull request merged**